app/controllers/reminder_mailings_controller.rb

```diff
@@ -0,0 +1,87 @@
+class ReminderMailingsController < ApplicationController
+  before_action :set_reminder_mailing, only: [:show, :edit, :update, :destroy, :initiate_mailing]
+
+  def index
+    authorize ReminderMailing
+    @reminder_mailings = ReminderMailing.order('created_at desc').all
```

Comment:
why do we need the ordering here instead of how we used it like before with a scope?